01-js-intro-and-jquery/.idea/.name

```diff
@@ -0,0 +1 @@
+01-js-intro-and-jquery
```

Comment:
**Minor tip:** try to exclude this directory with [gitignore](http://git-scm.com/docs/gitignore). ;)